test/blackbox-tests/test-cases/install/install-glob/install-glob-relative.t

```diff
@@ -0,0 +1,33 @@
+Install a glob pattern from a relative path
```

Comment:
Can you add to this description explaining the problem this causes. Copying the commit message here will be fine.